cmd/main.go

```diff
@@ -28,9 +28,9 @@ import (
 
 	"github.com/erikdubbelboer/gspt"
 	"github.com/google/uuid"
+	"github.com/grafana/pyroscope-go"
```

Comment:
May need to add `	_ "github.com/grafana/pyroscope-go/godeltaprof/http/pprof"`

<img width="1411" alt="image" src="https://github.com/juicedata/juicefs/assets/31313340/704b31df-3545-4359-9d6d-27ecc1eabac2">

https://grafana.com/docs/pyroscope/latest/configure-client/grafana-agent/go_pull/

Reply:
Done in 8bf5f5eab5e38f5777db4b6c990fd29d36d8f922.

Should it be added also in other places that import `_ "net/http/pprof"`?

Reply:
`cmd/main.go` is the unified entry point for the juicefs program, so I don't think it needs to be added anywhere else.